source/_includes/puppet-collections/_pc_versions.html

```diff
@@ -0,0 +1,12 @@
+<table>
+  <thead>
+    <tr>
+      <th>Puppet Collection</th> <th>puppet-agent</th> <th>puppetserver</th> <th>puppetdb</th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr><td><a href="/puppet/4.3/reference/puppet_collections.html#PC-1">PC 1</a></td> <td><a href="/puppet/4.3/reference/about_agent.html">1.3.2</a></td> <td><a href="/puppetserver/2.2/release_notes.html">2.2.0</a></td> <td><a href="/puppetdb/3.2/">3.2</a></td></tr>
+  </tbody>
+</table>
+
```

Comment:
Hmm, okay. I'll mull this over some more, but my immediate reaction is that we should be listing what's in the collection in a fuzzier way, especially since it still contains historical packages for all the versions we've ever shipped in that collection. 

So something more like: PC 1 contains "puppet-agent 1.0.0 and up", and we can seal that off with an upper bound once the collection is done... which won't be for quite a while.